src/common/commonutils/Internal.h

```diff
@@ -37,4 +37,6 @@
 
 #define PLAIN_STATUS_FROM_ERRNO(a) ((0 == a) ? "passed" : "failed")
 
+#define INT_ENOENT -999

```

Comment:
Is there any particular reason for `-999`? Could it be `INT_MIN`?

Reply:
It's the error code and for us means "not found", or why INT_*ENOENT*. INT_MIN won't do it, there may be values set at just that.